DbMocker/MockTableExtensions.cs

```diff
@@ -0,0 +1,98 @@
+﻿using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+
+namespace Apps72.Dev.Data.DbMocker
+{
+    public static class MockTableExtensions
+    {
+        private static readonly ConcurrentDictionary<Type, (string[], PropertyInfo[])> ColumnMetaStore
+               = new ConcurrentDictionary<Type, (string[], PropertyInfo[])>();
+
+        /// <summary>
+        /// Convert object to MockTable
+        /// </summary>
+        /// <param name="obj">source object</param>
+        /// <returns>MockTable with a single row</returns>
+        public static MockTable ToMockTable<T>(this T obj)
```

Comment:
These extension methods aren't a better solution, but I can improve their in the next PR

Reply:
Thanks for this improvement... Can you create another PR with only this feature?